src/extension.ts

```diff
@@ -1,6 +1,9 @@
 import * as vscode from 'vscode';
 import * as path from 'path';
 import { LocalHistoryManager } from './local-history/local-history-manager';
+import { LocalHistoryTreeProvider } from './local-history/local-history-tree-provider';
+// eslint-disable-next-line no-unused-vars
```

Comment:
I think we can remove this rule, it looks problematic and a bit buggy.
It is also not present in the main repository and likely for good reason.